### ISSUE TITLE
Paywall policy

### ISSUE THREAD

**Issue by dionyziz** [decrypto-org/blockchain-papers#98]
How do we feel about papers behind paywalls? Do we want to link to paywalls? Do we want to update our policy so that we mandate every paper we link to is available openly?

**Comment by OrfeasLitos** [comment]
I'm against links to paywalls. If a published paper is behind a paywall, I support linking to an open version of it (e.g. author's website). Let's hear more opinions before updating the policy though.

**Comment by dionyziz** [comment]
My opinion is that we should forbid links to paywalled sites completely. Only papers that have an open version available should be listed in our curated list, and only when the link provided is to the open version. The author's website is one option; ePrint and arXiv are another.

**Comment by montyly** [comment]
Alternatively, a paper can have two links, one to the processing/original version (which could be behind a paywall), and one to a public (potentially updated) version.

I do agree that a paper should always have a public link.

**Comment by OrfeasLitos** [comment]
I agree with @dionyziz in boycotting paywalled sites. I agree with @montyly in linking to 2 versions. See #99 for a discussion on the latter.

**Comment by dionyziz** [comment]
I don't mind providing two versions. Sounds good. If @dimkarakostas is also in agreement we can incorporate this policy to our contributions guideline.

The question that remains is for the formatting of these two links.

**Comment by dimkarakostas** [comment]
I'm fine with the two links.

**Comment by cnasikas** [comment]
I am also fine with the two links.
